### ISSUE TITLE
eruption and pipewire

### ISSUE THREAD

**Issue by Glog78** [X3n0m0rph59/eruption#53]
**Describe the bug**
Not a bug / feature request

**To Reproduce**
none

**Expected behavior**
I would love to use audio viz on distributions with pipewire

**Logs**
none

As far as i see currently pulseaudio and the root "workaround" is requiered to get the audio setup done. Could you check if it could also work with a local pulse server (localhost:port) which can be provided with pipewire too. Additional config option maybe ? Needs maybe to check periodically if the server is started or something like this since there pipewire daemon in the user context.

**Comment by X3n0m0rph59** [comment]
@Glog78 Thanks for posting this feature request!

I have looked into adding support for PipeWire and added some preliminary documentation on how to set up audio for PipeWire enabled Linux distributions.

I got it working using the PulseAudio interface of PipeWire on `ArcoLinux` (Arch based) using the packages `pipewire-git` and  `pipewire-pulse-git`. I only tried the unix domain socket transport, not the IP socket based transport, but I suppose that would work too.
